keycloak/variables.tf

```diff
@@ -0,0 +1,83 @@
+variable "argocd" {
+  type        = map(string)
+  description = "A set of values for enabling deployment through ArgoCD"
+  default     = {}
+}
+
+variable "conf" {
+  type        = map(string)
+  description = "A custom configuration for deployment"
+  default     = {}
+}
+
+variable "namespace" {
+  type        = string
+  default     = ""
+  description = "A name of the existing namespace"
+}
+
+variable "namespace_name" {
+  type        = string
+  default     = "keycloak"
+  description = "A name of namespace for creating"
+}
+
+variable "module_depends_on" {
```

Comment:
after Terraform update 0.14 this variable deprecated - use depends_on in module level